invenio_search_ui/static/templates/invenio_search_ui/togglebutton.html

```diff
@@ -0,0 +1,4 @@
+<select ng-click="handleClick()">
```

Comment:
it should be a model (I will make a pr on your instance), otherwise it adds `undefined=undefined` parameter to the url